src/templates/partials/menu.hbs

```diff
@@ -18,8 +18,8 @@
 
     <h2>Hannah Sutton Design</h2>
     <p>Melbourne, Australia</p>
-    <a href="#">create@hannahsuttondesign.com</a>
-    <a href="#">m +61 (0) 428 226 905</a>
+    <a href="mailto:create@hannahsuttondesign.com">create@hannahsuttondesign.com</a>
+    <a href="tel:+61428226905">m +61 (0) 428 226 905</a>
```

Comment:
Same here, `target="_blank"` because some phone apps / extentions take over the web page. We don't want that.